appveyor.yml

```diff
@@ -7,7 +7,7 @@ configuration:
 
 before_build:
   # - sh: sudo apt-get install libvulkan-dev
-  - sh:  sudo apt-get --yes install libgl-dev
+  - sh:  sudo apt-get --yes install libgl-dev libvulkan-dev
```

Comment:
Was this a reason for weird link/rpath error lately on appveyor?
I've tried many times to reproduce it locally on Ubuntu18 virtual machine, with no success.

Reply:
Yeah. Not sure if the SDK was supposed to provide the .so somehow but installing to the system works for sure.